.git-blame-ignore-revs

```diff
@@ -6,3 +6,5 @@
 59e8c71786fd1c154904b3fefa7d26d88f4d92d2
 # Change formatting (#10180)
 062623438b5dfd66682a967edc7b7c91bd29e888
+# Prettier config update
```

Comment:
This won't work because of the squash, you have to do it in two steps.

Reply:
I'll convert the team away from squash merging one of these days 😅